### PR TITLE
Documentation updates for distribution and staking precompiles

### PR DESCRIPTION
**Pull request by dssei**
## What is the purpose of the change?
Updates documentation for distribution and staking precompiles

## Describe the changes to the documentation

- Add docs for distribution `withdrawMultipleDelegationRewards` transaction and `rewards` query
- Add docs for staking `delegation` query

### CODE REVIEW

File: pages/dev-interoperability/precompiles/staking.mdx [hunk 1]
```diff
@@ -47,6 +47,37 @@ This precompile allows EVM contracts to interact with Sei's staking module, enab
   ) external returns (bool success);
   ```
 
+### Queries
+- `delegattion`: Queries delegation for a given delegator and validator address
```
Comment (codebycarson) [inline pages/dev-interoperability/precompiles/staking.mdx]: Typo here

Comment (dssei) [reply pages/dev-interoperability/precompiles/staking.mdx]: fixed

**Review by codebycarson — APPROVED** [review]

**Comment by cordt-sei** [comment]
Do we want this merged?

**Comment by dssei** [comment]
> Do we want this merged?

We, do once chain changes hit the main net.

**Pull request merged**